WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -3,10 +3,18 @@ class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
   def index
+    @everything_good = true
+
     @jobs_that_should_have_run_by_now = Delayed::Job.where(attempts: 0).where('created_at < ?', MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING.minutes.ago)
     @oldest_job_that_should_have_run_by_now = @jobs_that_should_have_run_by_now.order(:created_at).first
+    if @oldest_job_that_should_have_run_by_now
+      @everything_good = false
+    end
 
-    @everything_good = @oldest_job_that_should_have_run_by_now.nil?
+    @regulations_load_error = Regulation.regulations_load_error
+    if @regulations_load_error
+      @everything_good = false
+    end
```

Comment:
Yes, of course. Off-by-one error.

`.blank?` is possible more robust than `.nil?`